app/models/video.rb

```diff
@@ -2,4 +2,6 @@ class Video < ApplicationRecord
   has_many :user_videos, dependent: :destroy
   has_many :users, through: :user_videos
   belongs_to :tutorial
+
+  validates :video_id, presence: true
```

Comment:
Is there a specific reason to validate the video_id? I was under the impression that it can't be a "thing" without an id anyway? Does this point some other Video object/resource?